services/ui-src/src/components/fields/ChoiceListField.tsx

```diff
@@ -171,18 +144,53 @@ export const ChoiceListField = ({
     }
   };
 
-  // update field values
+  const getNestedFields = (choices: FieldChoice[]) => {
```

Comment:
Note: This function compiles the field info needed for all the nested child fields to be blanked on autosave. It checks the parent choices to see which ones are unchecked, then dives into each of those and blanks all nested children, regardless of their previous values (including if they are radio/checkbox, it doesn't care what choices are checked/unchecked -- everything gets blanked). All that field info is returned in an array to the autosave handler to be called there.